### ISSUE TITLE
remove packr dependency

### ISSUE THREAD

**Issue by phanirithvij** [amlwwalker/got-qt#10]
Use the official go embed and remove the packr dependency?

**Comment by amlwwalker** [comment]
Yes good thinking. I won't get around to it any time soon, but please make a PR and I will review and merge
